core/src/main/resources/error/README.md

```diff
@@ -79,16 +79,177 @@ The message format accepts string parameters via the C-style printf syntax.
 The quality of the error message should match the
 [guidelines](https://spark.apache.org/error-message-guidelines.html).
 
-Invariants:
+#### Invariants
 
 - Unique
 
 ### SQLSTATE
 
 SQLSTATE is an optional portable error identifier across SQL engines.
 For consistency, Spark only sets SQLSTATE as defined in the ANSI/ISO standard.
-Spark does not define its own classes or subclasses.
+SQLSTATE comprises a 2-character class value followed by a 3-character subclass value.
+Spark only uses the standard-defined classes and subclasses, and does not use implementation-defined classes or subclasses.
 
-Invariants:
+#### Invariants
 
 - Consistent across releases
+
+#### ANSI/ISO standard
+
+The following SQLSTATEs are from ISO/IEC CD 9075-2.
+
+|SQLSTATE|Class|Condition                                                   |Subclass|Subcondition                                                   |
+|--------|-----|------------------------------------------------------------|--------|---------------------------------------------------------------|
+|07000   |07   |dynamic SQL error                                           |000     |(no subclass)                                                  |
```

Comment:
no biggie but should we capitalize it? I don't know. will leave it to you.

Reply:
I don't particularly care either way. Given that this is pulled directly from the SQL manual and this is the style they went with, I'm ok with keeping it as is.